src/main/webapp/index.html

```diff
@@ -26,9 +26,9 @@
         <button id="select-text">Text</button>
         <button id="select-blob">Upload Image</button>
       </div>
-      <form id="keyword-form" class="hide" action="/keyword" method="post" enctype="multipart/form-data">
+      <form id="keyword-form" class="hide" action="/keyword" method="post" enctype="multipart/form-data" onsubmit="validateForm(event);">
         <textarea id="text-input" name="keyword-sentence" placeholder="Enter your tweet or interests here"></textarea>
-        <input id="blob-input" type="file" name="image" accept="image/*" />
+        <input id="blob-input" type="file" name="image" accept="image/png, image/jpeg, image/jpg" />
```

Comment:
Why are you only allowing these formats of images?

Reply:
I'm not sure actually lol, I just kinda changed it out of whim. I will change it back.